android/src/main/kotlin/io/xdea/flutter_vpn/FlutterVpnPlugin.kt

```diff
@@ -99,23 +100,35 @@ class FlutterVpnPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
 
   override fun onMethodCall(@NonNull call: MethodCall, @NonNull result: Result) {
     when (call.method) {
+      "hasPrepared" -> {
```

Comment:
Added this here, a return value of null from L104 means the VPN permissions have been accepted